docs/parameters.md

```diff
@@ -1,10 +1,53 @@
-# Using Parameters
+# Parameters
 
-Experiment parameters define the search space for assigned values that vary for each trial run. Each parameter represents a named integer assignment with an inclusive minimum and maximum bound.
+A **parameter** is a value which can be adjusted during an experiment. Every parameter has a name and a **domain** of acceptable values it can be **assigned** during a trial.
```

Comment:
> A **parameter** is a value which

Other things I considered besides "value": "field", "placeholder", "input to an experiment". I'm open to other options.

Reply:
I personally like the idea of variable vs. outcome or input vs. output, where variable/output=metrics

Reply:
Why so much emphasis?

Reply:
@stephanie-rifai Those are helpful, thanks. Think I might switch to input (Though I think you meant *output*/outcome=metrics?)

@jgustie I added emphasis each time a new term was introduced to improve readability. Though it's over-the-top in this sentence because of the emphasis on `assigned`, which a) makes three emphasized terms in one sentence, which is a lot, and b) doesn't look like a technical term. I'll split it out into another sentence.

Reply:
yes, sorry!